OoTClient.py

```diff
@@ -179,6 +179,12 @@ async def parse_payload(payload: dict, ctx: OoTContext, force: bool):
     locations = payload['locations']
     collectibles = payload['collectibles']
 
+    # The Lua JSON library serializes an empty table into a list instead of a dict. Verify types for safety:
+    if isinstance(locations, list):
+        locations = {}
```

Comment:
doesn't that delete the content though?

Reply:
we only get a list back if the table was completely empty, so there was no content anyway. It's sort of a workaround for a bug where an empty table gets sent when no table is supposed to be sent at all. I observed this only twice and was unable to replicate it using the same settings + different seed, but twice is enough for it to potentially happen again, so I decided to fix the issue here instead.